frontend/app/.server/express-server/opentelemetry.server.ts

```diff
@@ -84,6 +83,15 @@ function getTraceExporter(): SpanExporter {
   };
 }
 
+/**
+ * Transforms a string into an ingeger.
+ * Will return undefined if the string can't be transformed.
+ */
+function toNumber(str?: string): number | undefined {
+  const num = parseInt(str ?? '');
```

Comment:
Use `Number.parseInt` and `Number.isNaN`.
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-number-properties.md

Reply:
plz re-review